### PR TITLE
Header help modal

### PR DESCRIPTION
**Pull request by BazilMuralidharan**
URL for testing:

- https://headerHelpModal--hdfc-bank--aemsites.hlx.page/

### CODE REVIEW

File: blocks/icon-modal/icon-modal.css [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+body.modal-open {
+  overflow: hidden;
+}
+
+.icon-modal dialog {
+  --dialog-border-radius: 16px;
+  
+  overscroll-behavior: none;
+  border: 1px solid #ccc;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:12]: use color code

File: blocks/icon-modal/icon-modal.css [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+  fieldset[class*='field-aadharconsent-'] {
+    padding: 0;
+
+    div[class*='field-aadharconsenttext-'] { 
+      padding: 20px 0 0 24px; 
+
+      p { 
+        color: var(--color-input-text-default); 
+        font-weight: 500;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:55]: use typography

File: blocks/icon-modal/icon-modal.css [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+.modal-content {
+  max-width: 920px;
+  width: 100%;
+   color: var(--color-modal-interface-text-primary);
+   overflow: auto;
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:82]: use typography

File: blocks/icon-modal/icon-modal.css [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+}
+
+.modal-content {
+  max-width: 920px;
+  width: 100%;
+   color: var(--color-modal-interface-text-primary);
+   overflow: auto;
+
+  p {
+    font-size: 14px;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:80]: use typography variables

File: blocks/icon-modal/icon-modal.css [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+    letter-spacing: 0; 
+    text-align: left;
+  }
+
+  legend{
+    display: none;
+  }
+
+  .field-label {
+    font-size: 18px;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:92]: use typography variables

File: blocks/icon-modal/icon-modal.css [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+  overflow-y: auto;
+  overscroll-behavior: none;
+  max-height: calc(95dvh - 60px);
+}
+
+@media (width >=768px) {
+  .icon-modal dialog {
+    width: 100%;
+    max-width: 480px;
+    font-size: 14px;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:144]: use typography

File: blocks/icon-modal/icon-modal.css [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+  right: 0;
+  width: 24px;
+  height: auto;
+  padding: 24px 50px 0 24px;
+  border-radius: 0 var(--dialog-border-radius) 0 0;
+  background-color: unset;
+  text-overflow: clip;
+  margin: 0;
+  border: none;
+  color: #6b7280;
```
Comment (kunal-adobe) [inline blocks/icon-modal/icon-modal.css:184]: use color code

File: blocks/icon-modal/icon-modal.css (hunk 2 above)
Comment (sureadobe24) [reply blocks/icon-modal/icon-modal.css]: resolves in both the files

File: blocks/icon-modal/icon-modal.css (hunk 3 above)
Comment (sureadobe24) [reply blocks/icon-modal/icon-modal.css:81]: resolved in both the files

File: blocks/icon-modal/icon-modal.css (hunk 5 above)
Comment (sureadobe24) [reply blocks/icon-modal/icon-modal.css]: resolved in both

File: blocks/icon-modal/icon-modal.css (hunk 7 above)
Comment (sureadobe24) [reply blocks/icon-modal/icon-modal.css]: resolved in both the files

File: blocks/icon-modal/icon-modal.css (hunk 6 above)
Comment (sureadobe24) [reply blocks/icon-modal/icon-modal.css]: resolved in both the places

**Review by kunal-adobe — APPROVED** [review]

**Pull request closed without merging**